Python/Text_Extract/text_extract.py

```diff
@@ -0,0 +1,43 @@
+from PIL import Image
+import pytesseract as pt
+import os
+from pathlib import Path
+
+
+current_location = (os.getcwd() + '\\')
+
+
+def main():
+    # Enter the name of folder which contains img files
+    image_location = input("Enter the Folder name containing Images: ")
+    image_path = os.path.join(current_location, image_location)
+    # Enter the name of folder which would contain respective txt files
+    destination = input("Enter your desired output location: ")
+    destination_path = os.path.join(current_location, destination)
+    # Path to Tesseract
+    pt.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract'
+
+    # iterating the images inside the folder
+    for imageName in os.listdir(image_path):
+        # Join the path and image name to obtain absolute path
+        inputPath = os.path.join(image_path, imageName)
+        img = Image.open(inputPath)
+
+        # OCR
+        text = pt.image_to_string(img, lang="eng")
+
+        # Removing extensions
+        img_file = Path(inputPath).stem
+        print(img_file)
+
+        # The output text file
+        text_file = img_file + ".txt"
```

Comment:
Can we use CSV instead of TXT? Just a general question.

Reply:
Saving text in CSV, Why?
I can do that but I don't understand why we need that

Reply:
Moreover, How you want that CSV to look like, Comma separating every line? 

But I believe that changes the reason why we prefer CSV, similar data separated by a comma, but considering lines can be really long or really small, I don't see the reason.

Reply:
Seems reasonable 👍